src/dataframe/dataframe.jl

```diff
@@ -1162,7 +1183,8 @@ end
 
 Base.convert(::Type{DataFrame}, A::AbstractMatrix) = DataFrame(A)
 
-Base.convert(::Type{DataFrame}, d::AbstractDict) = DataFrame(d)
+Base.convert(::Type{DataFrame}, d::AbstractDict; copycolumns::Bool=true) =
```

Comment:
Do we want to allow `copycolumns` kwarg here?

Reply:
Maybe not. `convert` is supposed to avoid copying AFAIK, so people should use the constructor if they don't like that.

Reply:
OK. Removed